static/app/views/explore/hooks/useTimeseries.tsx

```diff
@@ -0,0 +1,137 @@
+import type {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+
+/**
+ * Row represents a point in a timeseries - plotting (timestamp, value) with a line between them would represent a line chart.
+ */
+export interface Row {
```

Comment:
This is more of a general note, but could you align this a little more with the existing `TimeSeries` types? e.g., this should be called `TimeSeriesItem`, `SeriesMeta` should be called `TimeSeriesMeta`, etc.